tests/sf-tests.xml

```diff
@@ -6,13 +6,19 @@
   colors                      = "true"
   processIsolation            = "true"
   bootstrap                   = "bootstrap-sf.php"
+  convertErrorsToExceptions="true"
+  convertNoticesToExceptions="true"
+  convertWarningsToExceptions="true"
 >
   <php>
     <server name="KERNEL_CLASS" value="AppKernel" />
     <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak" />
```

Comment:
maybe we can remove the `SYMFONY_DEPRECATIONS_HELPER` then?

Reply:
Not sure it's worth it, but if we do it it should be done in another pull request because it changes the behavior.

What I do here: tracking of all PHP errors,
What the Symfony deprecations helpers do: highlight every deprecation (but mostly the Symfony one).

I'm not against making PrestaShop compatible with Symfony 4, but it will create a lot of "noise" right now when our test suite have already a lot of noise for reasons we don't even understand.

Reply:
alright let's keep this for later